index.md

```diff
@@ -0,0 +1,32 @@
+# Protractor helper's library
+
+[![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) [![npm weekly downloads](https://img.shields.io/npm/dw/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) 
+
+This library contains helper functions that can be used together with Protractor for creating robust end-to-end tests.
+
+Many of the helper functions on this library uses `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps on avoiding trying to interact with elements when they are still not ready for it, which helps on avoiding test flakiness.
+
+## Installation
+
+Below it is described the process of Installation of such module.
```

Comment:
Shorten sentence.
What about:
`Below is described the library's installation process.`

Reply:
Done.